cityofaustin/janis
# 3575 - change email to participate in the feedback error message

# Description

Updated the feedback email to the correct one, participate@austintexas.gov








# Types of changes



# How can this be tested?



 

To test, go to submit feedback from any page, because of CORS the error module should show up. check that the email address is participate and that think link is also correct. 


# Checklist:
